xtask/src/build_docker_image.rs

```diff
@@ -11,6 +11,8 @@ pub struct BuildDockerImage {
     ref_type: Option<String>,
     #[clap(long, hide = true, env = "GITHUB_REF_NAME")]
     ref_name: Option<String>,
+    #[clap(long, hide = true, env = "IS_LATEST")]
+    is_latest: Option<bool>,
```

Comment:
Maybe a better idea would be to make this a `bool` and have it be called `latest`, and access the `LATEST` envvar? `--latest` or `--set-latest` would describe an action seems better.

Reply:
fixed, kept the name `is_latest` but flag is --latest

Reply:
FYI: you've changed the envvar name but I still think `is_latest` for the CLI should be renamed to `latest`. Going to test it out shortly but looks good at first glance.

Reply:
it is renamed, `long = "latest"`